web/ui/src/Hosts/HostsController.js

```diff
@@ -127,16 +127,19 @@
             // update hosts
             update();
 
-            servicesFactory.activate();
             hostsFactory.activate();
             poolsFactory.activate();
+            // need instances to roll up
+            // RAM commitment per host
+            instancesFactory.activate();
```

Comment:
need instances so that hosts can rollup ram usage. Ideally the server would do this calculation.